internal/filesystem/ipfs/ipns.go

```diff
@@ -63,6 +64,7 @@ func NewIPNS(core coreiface.CoreAPI, ipfs fs.FS, options ...IPNSOption) (*IPNS,
 					readAll | executeAll,
 			},
 			nodeTimeout: 1 * time.Minute,
+			linkLimit:   40, // Arbitrary.
```

Comment:
raise to constant somewhere?

Reply:
For sure. This is part of a larger problem as well, detailed below but can be ignored.

---

The way I handle mountpoint data (the parameters required to instantiate a file system) is currently weird.
And I was thinking about better ways to handle them.

The `ipfs` pkg has constructors for the IPFS `fs.FS` systems, but in `/command/mount` we create a struct containing values for that constructor.
(The mount command is just a client that sends that struct + other data to the `fs` daemon process, which translates that portion back into parameters for the `fs.FS` constructor which is calls in its own process space).

I'm putting thought into how to better deal with this, but I'm not sure yet.
One thing I was considering was just changing the constructors to accepting a settings struct, and exposing that. This way the constructor parameter is itself an exposed serializable type/format.
The other was going to be to keep them as-is, but expose all the default values in exported consts `DefaultLinkLimit = 40`, so that external packages could come up with their own serializable format if they need to, irrelevant of how the Go API is.
Or some other nonsense.

The main reason the constructor options need to be serializable, is so that we can implement something similar to Unix's `fstab`. We can just dump the current mountpoints to disk, and load them again at startup.
(This currently works, but there's no command to automate it. I want to fix this up first before doing that.)